src/app/about/about.component.html

```diff
@@ -1,3 +1,3 @@
-<p>
-  about works!
-</p>
+<div align="center">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "align" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3AS1827)